lib/SIL/SILInstructions.cpp

```diff
@@ -2496,10 +2524,27 @@ GraphOperationInst *GraphOperationInst::create(
   }
 
   unsigned size =
-    totalSizeToAlloc<MultipleValueInstruction *, GraphOperationResult, Operand,
-                     GraphOperationAttribute>(
-      1, resultTypes.size(), arguments.size(), attributes.size());
+    totalSizeToAlloc<MultipleValueInstruction *, GraphOperationResult, Operand>(
+      1, resultTypes.size(), arguments.size());
   void *buffer = M.allocateInst(size, alignof(GraphOperationInst));
   return ::new (buffer) GraphOperationInst(M, loc, name, arguments, attributes,
                                            resultTypes, resultOwnerships);
 }
+
+ArrayRef<GraphOperationAttribute>
+GraphOperationInst::getAttributes() const {
+  return Attributes;
+}
+
+MutableArrayRef<GraphOperationAttribute>
+GraphOperationInst::getAttributes() {
```

Comment:
the getAttributes() calls can stay inline, but getAttribute(i) needs to be out of line.

Reply:
Done, thanks.